android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/SignTransactionFragment.java

```diff
@@ -203,6 +209,11 @@ private void updateSignRequestData() {
         updateActionState(mTxRequestNumber == 0);
         updateAccount(fromAddress);
         updateNetwork(coin);
+        if (originInfo != null && URLUtil.isValidUrl(originInfo.originSpec)) {
```

Comment:
**originInfo.originSpec**
is this always not null ?

Reply:
no: https://github.com/brave/brave-core/blob/5359a4377666706adc7bd31c7610d57d7117cda5/android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/SignTransactionFragment.java#L172